### PR TITLE
Fix type error on build

### PR DESCRIPTION
**Pull request by ghost**
That should fix the error message as detailed in the README file :+1:

### CODE REVIEW

File: src/GreenText.tsx [hunk 1]
```diff
@@ -1,7 +1,9 @@
 import m from 'mithril';
 
-export default function(): m.Component {
-    return {
-        view: (vnode: m.Vnode) => <span style='color: green'>{vnode.children}</span>
-    };
+export default class implements m.ClassComponent {
```
Comment (tmj-code) [inline src/GreenText.tsx:3]: This still gives an error when I run the command 'npm run build'. But thanks for the suggestion.

Comment (ghost) [reply src/GreenText.tsx:3]: Well, this is the output of my successful build:
```
npm run build

> mithril-vite-tsx@0.0.0 build
> tsc && vite build

vite v2.8.2 building for production...
✓ 53 modules transformed.
dist/assets/favicon.17e50649.svg   1.49 KiB
dist/index.html                    0.49 KiB
dist/assets/index.5b4933c6.js      0.90 KiB / gzip: 0.52 KiB
dist/assets/index.06d14ce2.css     0.17 KiB / gzip: 0.14 KiB
dist/assets/vendor.fe47eba3.js     27.38 KiB / gzip: 10.03 KiB
```

Comment (tmj-code) [reply src/GreenText.tsx:3]: Well, I stand corrected. After i cleared node_modules and reinstalled packages, the ClassComponent change did the trick.  The function approach still does not work, but that is okay. Thank you very much.

Comment (ghost) [reply src/GreenText.tsx:3]: Awesome!

From [here](https://github.com/MithrilJS/mithril.d.ts/blob/84a87fefe18868c1b56a0a9a09efc7443a1fbd0f/readme.md#jsxtsx) I can read:
> Library support is required for full TSX support and cannot be accomplished with types alone. See the NPM package [mithril-tsx-component](https://www.npmjs.com/package/mithril-tsx-component).

This archived repository has an issue related to yours: https://github.com/andraaspar/mithril-tsx-component/issues/2

Comment (tmj-code) [reply src/GreenText.tsx:3]: Yes, it appears to be the same problem. I am not sure about the library support thing - TSX seems to work without it. But perhaps I have just not required the 'full' TSX support yet. However, what I have is working, so I am happy.

**Pull request merged**